spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Cell.scala

```diff
@@ -36,6 +36,8 @@ abstract class Cell[T : ClassTag](val hiddensShape: Array[Int])
   (implicit ev: TensorNumeric[T])
   extends AbstractModule[Table, Table, T] {
 
+  var cell: AbstractModule[Activity, Activity, T]
```

Comment:
Usually cell is a Sequential. The `Sequential` extends` Container[Activity, Activity, T]`, which is not a subclass of `AbstractModule[Table, Table, T]`. So if I set cell to AbstractModule[Table, Table, T], there will be a compile error.